src/rpc/channels.ts

```diff
@@ -46,7 +62,10 @@ export type BrowserInitializer = {};
 
 export interface BrowserContextChannel extends Channel {
   on(event: 'bindingCall', callback: (params: BindingCallChannel) => void): this;
+  on(event: 'close', callback: () => void): this;
   on(event: 'page', callback: (params: PageChannel) => void): this;
+  on(event: 'route', callback: (params: { route: RouteChannel, request: RequestChannel }) => void): this;
```

Comment:
Why do we have route event on both Page and BrowserContext?

Reply:
Let me see.